src/views-components/main-app-bar/main-app-bar.tsx

```diff
@@ -34,6 +35,7 @@ export interface MainAppBarActionProps {
     onSearch: (searchText: string) => void;
     onBreadcrumbClick: (breadcrumb: Breadcrumb) => void;
     onMenuItemClick: (menuItem: MainAppBarMenuItem) => void;
+    onDetailsPanelClick: (isOpened: boolean) => void;
```

Comment:
Imo `onDetailsPanelToggle` would be a better name in this case and the callback should not have any parameter. Parent component should decide what happen when you call it.